utils/funcexp/func_least.cpp

```diff
@@ -22,6 +22,10 @@
 *
 ****************************************************************************/
 
+#include <mariadb.h>
```

Comment:
What do you think if we add this into a common header instead?

Reply:
I think it probably needs to be done.